Makefile

```diff
@@ -10,6 +10,10 @@ ifndef CN
 CN := $(shell hostname)
 endif
 
+ifndef CRL
+CRL := "http://example.com/fake.crl.pem"
```

Comment:
This will still always specify a CRL. I would like the default to be no CRL, and no "fake" CRL url.